src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,18 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.15.2 (July 4, 2023)
+
+* The `subctl cloud prepare azure` command has a new flag, `air-gapped`, to indicate the cluster is in an air-gapped
```

Comment:
```suggestion
* The `subctl cloud prepare azure` command has a new flag, `air-gapped`, to indicate the cluster is in an air-gapped.
```

Reply:
The sentence continues on the next line.